cobbler/actions/status.py

```diff
@@ -44,6 +44,11 @@ class CobblerStatusReport(object):
     def __init__(self, collection_mgr, mode, logger=None):
         """
         Constructor
+
+        :param collection_mgr: The collection manager which holds all information.
+        :param mode: This describes how Cobbler should report. Currently there can be explicitly only the option
```

Comment:
I would rewrite this. Something like

"Currently only the option ``text`` can be set explicitly".